src/components/DatePicker.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from "react";
+import DateBulletItem from "@/commons/DateBulletItem";
+import IconButton from "@/commons/IconButton";
+import { RiArrowLeftSLine, RiArrowRightSLine } from "react-icons/ri";
+import { BsCalendarEvent } from "react-icons/bs";
+
+export interface DatePickerProps {
+  className?: string;
+}
+
+const DatePicker: FC<DatePickerProps> = ({ className }) => {
+  const date = new Date();
+  const month = date.toLocaleString("UTC", { month: "long" });
+  const years = date.getFullYear();
+  const currentDate = date.toDateString();
+  const daysMonth = new Date(years, date.getMonth() - 1, 0).getDate();
+  const diasSemana = ["Dom", "Lun", "Mar", "Mie", "Jue", "Vie", "Sáb"];
+  const dateMonth = [];
+
+  for (let day = 1; day <= daysMonth; day++) {
+    const indice = new Date(years, date.getMonth(), day).getDay();
```

Comment:
Pequeñísimo detalle:
```suggestion
    const index= new Date(years, date.getMonth(), day).getDay();
```